python/docs/source/user_guide/sql/python_udtf.rst

```diff
@@ -163,6 +185,28 @@ To implement a Python UDTF, you first need to define a class implementing the me
             ...         num_articles=len((
             ...             word for word in words
             ...             if word == 'a' or word == 'an' or word == 'the')))
+
+            An `analyze` implementation that returns a constant output schema, and also requests
+            to select a subset of columns from the input table and for input table to be partitioned
+            across several UDTF calls based on the values of the `date` column:
+
+            >>> @staticmethod
+            ... def analyze(*args) -> AnalyzeResult:
```

Comment:
It will be very helpful to have some explanation for this function (what this UDTF is trying to achieve with partitionBy, orderBy, and select). Maybe we can add it in the example section.